airbyte-cdk/python/airbyte_cdk/sources/streams/http/http.py

```diff
@@ -64,18 +62,15 @@ def use_cache(self) -> bool:
         return False
 
     def request_cache(self) -> requests.Session:
-        self.clear_cache()
-        return requests_cache.CachedSession(self.cache_filename)
+        backend = requests_cache.SQLiteCache(use_temp=True)
+        return requests_cache.CachedSession(self.cache_filename, backend=backend)
 
     def clear_cache(self) -> None:
```

Comment:
is this method still used? If not, can we delete it?

Reply:
I think if we have some kind of caching mechanism it is good to have a way to force cleaning of the cache. At the moment the only place where we use it is unit tests to simulate fresh start of the stream.